server.js

```diff
@@ -27,15 +27,15 @@ app.get('/api/hello', function(req, res) {
 });
 
 app.post('/api/shorturl', (req, res) => {
-  const { original_url } = req.body
+  const { url } = req.params
```

Comment:
This should be req.body, was trying req.params but forgot to change it back. In next commit it had been changed to body.